src/BullOak.Repositories.EventStore.Test.Integration/Specification/SaveEventStream.feature

```diff
@@ -134,3 +134,27 @@ Examples:
 	| 10           | 3            | 2         |
 	| 10           | 10000        | 9999      |
 	| 10000        | 10           | 9         |
+
+Scenario: When invariants fail no events are saved
+	Given an always-fail invariant checker
+	And an existing stream with 3 events
+	And 5 new events
+	When I try to save the new events in the stream
+	And I load my entity ignoring any errors
```

Comment:
> `And I load my entity ignoring any errors`

does this belong to `WHEN`? If yes, it is not clear why - it is mixing save and load actions

If no, maybe move this to `GIVEN`?

```gherkin
Scenario: When invariants fail no events are saved
	Given an always-fail invariant checker
	And an existing stream with 3 events
	And 5 new events
	And I ignore any errors on load
	When I try to save the new events in the stream
...
```

Reply:
We can't do _that_ because we need it for validations _after_ we tried to save the events and it's core to the test.

I get your point, but the only option is to move it in the `Then` clauses, which makes it even weirder imho. What do you think?

Reply:
OK then. I guess better leave as is.